src/servers/main.cc

```diff
@@ -324,6 +325,9 @@ std::vector<Option> options_
       {OPTION_BACKEND_DIR, "backend-directory", Option::ArgStr,
        "The global directory searched for backend shared libraries. Default is "
        "'/opt/tritonserver/backends'."},
+      {OPTION_ASYNC_WORKER_COUNT, "async-worker-count", Option::ArgInt,
```

Comment:
Right now we are using a singleton async work class just for buffer copy acceleration but in the future we may have more than one worker pool for different uses. So let's make the flag specific to the current use case. Use --buffer-manager-thread-count and a description like. "The number of threads used to accelerate copies and other operations required to manage input and output tensor contents. Default is 1."  We can correctly handle 0 value but it is confusing to make that the default since it implies no threads are doing copies.

Also adjust naming for related API additions.